examples/testing/main_test.sh

```diff
@@ -16,3 +16,29 @@ $ go test -v
     --- PASS: TestIntMinTableDriven/-1,0 (0.00s)
 PASS
 ok  	examples/testing	0.023s
+
+# Run all benchmarks in the current project in verbose
+# mode. All tests are run prior to benchmarks. The `bench`
+# flag receives a regex for benchmark function names.
+$ go test -v -bench=.
+=== RUN   TestIntMinBasic
```

Comment:
I would leave the `-v` out of the benchmark example - it just repeats lots of long text

Reply:
Resolved in bf78b89